src/checks.rs

```diff
@@ -53,6 +53,10 @@ pub enum CheckCode {
     F406,
     F407,
     F521,
+    F522,
+    F523,
+    F524,
+    F525,
```

Comment:
Can you run `cargo dev generate-rules-table`? It'll add these to the README automatically.